### PR TITLE
Fix org.restlet and org.jacoco dependencies

### PR DESCRIPTION
**Pull request by smatzana**

### CODE REVIEW

**Comment by smatzana** [comment]
Evaluating your uReplicator but had to make these changes in order to build from master

**Comment by xhl1988** [comment]
lgtm

File: pom.xml [hunk 1]
```diff
@@ -105,6 +105,16 @@
 		<junit.version>4.6</junit.version>
 	</properties>
 
+	<repositories>
```
Comment (xhl1988) [inline pom.xml]: Could you add this in the uMirrorMaker-Controller/pom.xml instead of root pom?

Comment (smatzana) [inline pom.xml]: ``uMirrorMaker-Worker`` needs it too, that's why I put it in the top level pom

**Comment by xhl1988** [comment]
Already merged #16

**Pull request closed without merging**